### PR TITLE
[CYB-135] Bash script update

### PR DESCRIPTION
**Pull request by vpavlenko-cv**

### CODE REVIEW

File: flink-cyber/cyber-jobs/src/main/resources/scripts/cs-upsert-rule
Comment (carolynduby) [inline flink-cyber/cyber-jobs/src/main/resources/scripts/cs-upsert-rule:1]: where is the run_java_class script defined?

**Review by carolynduby — COMMENTED** [review]
Hi Vlad - 

looks good but I'm not sure about the upsert rule script.  Where is the run_java_class command defined?

File: flink-cyber/cyber-jobs/src/main/resources/scripts/cs-add-enrich
Comment (carolynduby) [inline flink-cyber/cyber-jobs/src/main/resources/scripts/cs-add-enrich:1]: I think this script needs to stay.  This is adding a new enrichment loading config file.

**Review by carolynduby — CHANGES_REQUESTED** [review]

File: flink-cyber/cyber-parcel/src/main/meta/permissions.json [hunk 1]
```diff
@@ -1,14 +1,4 @@
 {
-  "bin": {
-    "user": "cybersec",
-    "group": "cybersec",
-    "permissions": "0755"
-  },
-  "bin/cs-add-enrich": {
```
Comment (carolynduby) [inline flink-cyber/cyber-parcel/src/main/meta/permissions.json:7]: Do we need to add this back because we added cs-add-enrich back in?

**Review by carolynduby — CHANGES_REQUESTED** [review]

File: flink-cyber/cyber-parcel/src/main/meta/permissions.json (hunk 1 above)
Comment (vpavlenko-cv) [reply flink-cyber/cyber-parcel/src/main/meta/permissions.json:7]: yes

**Review by carolynduby — APPROVED** [review]
Looks good.

**Pull request merged**